File: Java/Product.java
public class Product{
   
    private int price;  //Deklarasi Atribut price
    private int idProduct;  //Deklarasi Atribut id prodak
    
    //Constructor
    public Product() {
    }

    
    // getter and setter
    public void setidProduct(int idProduct) {
        this.idProduct = idProduct;
    }
    public int getidProduct() {
        return idProduct;
    }

    public void setPrice(int price) {
        this.price = price;
    }
    public int getPrice() {
        return price;
    }

    //Output 
    public void output_product(){
        System.out.println("Id Product : "+ this.getidProduct());
        System.out.println("Price : "+ this.getPrice());
    }

}

File: C++/Product.cpp
#include <string>

using namespace std;

class Product
{
private:
	int idProduct;
	int price;

public:
	//Constructor 
	Product(int idProduct, int price)
	{
		this->idProduct = idProduct;
		this->price = price;
	};
    
    //Constructor 
    Product(){

	};

	//get set
	void setIdProduct(int idProduct)
	{
		this->idProduct = idProduct;
	};

	int getIdProduct()
	{
		return this->idProduct;
	};

	void setPrice(int price)
	{
		this->price = price;
	};

	int getPrice()
	{
		return this->price;
	}
};